### PR TITLE
Amend PR checklist to check for hot fixes to revert (#3115)

### PR DESCRIPTION
**Pull request by jessebrennan**
Author

- [ ] PR title references issue
- [ ] Title of main commit references issue
- [ ] PR is linked to Zenhub issue

Author (reindex)

- [ ] Added `r` tag to commit title                         <sub>or this PR does not require reindexing</sub>
- [ ] Added `reindex` label to PR                           <sub>or this PR does not require reindexing</sub>

Author (freebies & chains)

- [ ] Freebies are blocked on this PR                       <sub>or there are no freebies in this PR</sub>
- [ ] Freebies are referenced in commit titles              <sub>or there are no freebies in this PR</sub>
- [ ] This PR is blocked by previous PR in the chain        <sub>or this PR is not chained to another PR</sub>
- [ ] Added `chain` label to the blocking PR                <sub>or this PR is not chained to another PR</sub>

Author (upgrading)

- [ ] Documented upgrading of deployments in UPGRADING.rst  <sub>or this PR does not require upgrading</sub>
- [ ] Added `u` tag to commit title                         <sub>or this PR does not require upgrading</sub>
- [ ] Added `upgrade` label to PR                           <sub>or this PR does not require upgrading</sub>

Author (requirements, before every review)

- [ ] Ran `make requirements_update`                        <sub>or this PR leaves requirements*.txt, common.mk and Makefile untouched</sub>
- [ ] Added `R` tag to commit title                         <sub>or this PR leaves requirements*.txt untouched</sub>
- [ ] Added `reqs` label to PR                              <sub>or this PR leaves requirements*.txt untouched</sub>

Author (before every review)

- [ ] `make integration_test` passes in personal deployment <sub>or this PR does not touch functionality that could break the IT</sub>
- [ ] Rebased branch on `develop`, squashed old fixups

Primary reviewer (after approval)

- [ ] Commented in issue about demo expectations            <sub>or labelled issue as `no demo`</sub>
- [ ] Decided if PR can be labeled `no sandbox`
- [ ] PR title is appropriate as title of merge commit
- [ ] Moved ticket to Approved column
- [ ] Assigned PR to an operator

Operator (before pushing merge the commit)

- [ ] Checked `reindex` label and `r` commit title tag
- [ ] Checked that demo expectations are clear              <sub>or issue is labeled as `no demo`</sub>
- [ ] Rebased and squashed branch
- [ ] Sanity-checked history
- [ ] Pushed PR branch to Github
- [ ] Branch pushed to Gitlab                               <sub>or PR is labeled `no sandbox`</sub>
- [ ] Build passes in sandbox and added `sandbox` label     <sub>or PR is labeled `no sandbox`</sub>
- [ ] Started reindex in `sandbox`                          <sub>or this PR does not require reindexing `sandbox`</sub>
- [ ] Checked for failures in `sandbox`                     <sub>or this PR does not require reindexing `sandbox`</sub>
- [ ] Added PR reference to merge commit title
- [ ] Collected commit title tags in merge commit title
- [ ] Moved linked issue to Merged column
- [ ] Pushed merge commit to Github

Operator (after pushing the merge commit)

- [ ] Moved freebies to Merged column                       <sub>or there are no freebies in this PR</sub> 
- [ ] Shortened the PR chain                                <sub>or this PR is not the base of another PR</sub>
- [ ] Verified that `N reviews` labelling is accurate
- [ ] Pushed merge commit to Gitlab                         <sub>or this changes can be pushed later, together with another PR</sub>
- [ ] Deleted PR branch from Github and Gitlab

Operator (reindex) 

- [ ] Started reindex in `dev`                              <sub>or this PR does not require reindexing or does not target `dev`</sub>
- [ ] Checked for failures in `dev`                         <sub>or this PR does not require reindexing or does not target `dev`</sub>
- [ ] Started reindex in `prod`                             <sub>or this PR does not require reindexing or does not target `prod`</sub>
- [ ] Checked for failures in `prod`                        <sub>or this PR does not require reindexing or does not target `prod`</sub>

Operator

- [ ] Unassigned PR

### CODE REVIEW

**Comment by jessebrennan** [comment]
@amarjandu in order for this checkbox to make sense, I had to add a few other checkboxes which describe the `prod` promotion. I'm not sure if these are too tedious or how they will work in practice, so I'm looking for feedback on this in your review.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -75,3 +75,12 @@ Operator (reindex)
 Operator
 
 - [ ] Unassigned PR
+
+---  
+
+Operator (prod)
+- [ ] Revert `prod` hotfixes for this PR                    <sub>or there are no hotfixes for this PR in `prod`</sub>
+- [ ] Merge `develop` into `prod`
```
Comment (amarjandu) [inline .github/pull_request_template.md:83]: I've not done a `prod` release; but do we also do a `prod` -> `dev` merge as well? Or am I imaging that I heard that in a discussion between @hannes-ucsc  and @noah-aviel-dove for the last hot-fix?


Also see:
1280674d4197ca5746895a0eeeee43f6b69a63a5

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -75,3 +75,12 @@ Operator (reindex)
...
+- [ ] Push `prod` to `github` and `gitlab`
```
Comment (amarjandu) [inline .github/pull_request_template.md:84]: Other references to Github/Gitlab are proper nouns and don't back-ticks.

**Review by amarjandu — CHANGES_REQUESTED** [review]
Might also want to  add into the `wiki` about what is involved with removing a hotfix from prod.

File: .github/pull_request_template.md (hunk 1 above)
Comment (jessebrennan) [reply .github/pull_request_template.md:83]: Good questions. I'll bring this up in PL.

**Comment by jessebrennan** [comment]
Closing because we would rather add this as a step in the operator manual.

**Pull request closed without merging**